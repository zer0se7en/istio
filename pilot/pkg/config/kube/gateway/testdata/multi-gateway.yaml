apiVersion: networking.x-k8s.io/v1alpha1
kind: GatewayClass
metadata:
  name: istio
spec:
  controller: istio.io/gateway-controller
---
apiVersion: networking.x-k8s.io/v1alpha1
kind: Gateway
metadata:
  name: gateway
  namespace: istio-system
spec:
  gatewayClassName: istio
  addresses:
  - type: NamedAddress
    value: istio-ingressgateway
  - type: NamedAddress
    value: istio-ingressgateway.not-default.svc.domain.suffix
  - type: NamedAddress
    value: example.com
  listeners:
  - hostname: "*.domain.example"
    port: 80
    protocol: HTTP
    routes:
      kind: HTTPRoute
  - port: 34000
    protocol: TCP
    routes:
      namespaces:
        from: All
      kind: TCPRoute
