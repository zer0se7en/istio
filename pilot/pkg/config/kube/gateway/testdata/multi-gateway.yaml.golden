apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: istio-ingressgateway.istio-system.svc.domain.suffix,istio-ingressgateway.not-default.svc.domain.suffix,example.com
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway
  namespace: istio-system
spec:
  servers:
  - hosts:
    - '*.domain.example'
    port:
      name: 0-gateway-gateway-istio-system
      number: 80
      protocol: HTTP
  - hosts:
    - '*'
    port:
      name: 1-gateway-gateway-istio-system
      number: 34000
      protocol: TCP
---
